extensions/ai-gen/src/components/CreateImageForm.tsx

```diff
@@ -35,6 +43,28 @@ export function CreateImageForm(props: { draftValues?: CreateImageValues }) {
     return true;
   }
 
+  function validateFile(images?: string[]) {
+    const [image] = images || [];
+    const stats = fs.statSync(image);
+    const size = sizeOf(image);
+
+    const isImage = !!image || images?.length === 1;
+    const isSquare = size.height == size.width;
+    const isCorrectSize = stats.size <= MAX_IMAGE_FILE_SIZE;
+    const isPng = path.extname(image).toLowerCase() === ".png";
+
+    if (!isImage || !stats.isFile() || !isSquare || !isCorrectSize || !isPng) {
+      setImageError("Image must be a square PNG and under 4MB");
```

Comment:
We were talking internally about updating the `FilePicker` API to only allow certain file types. So hopefully you wont have to check if it's `.png` in here anymore. 

BTW, how has your experience been using the `FilePicker` Component?

Reply:
Smooth sailing, honestly. My only gripe is I wish it returned a string instead of an array of strings if you set it to only allow a single file.